maven-embedder/src/main/java/org/apache/maven/cli/MavenCli.java

```diff
@@ -1194,7 +1194,7 @@ else if ( userSuppliedConfigurationProcessorCount > 1 )
                     sb.append( String.format( "%s\n", configurationProcessor.getClass().getName() ) );
                 }
             }
-            sb.append( String.format( "\n" ) );
+            sb.append( "\n" );
```

Comment:
this should be platform specific

Reply:
few lines above we also have some \n and then logic in logSummary -IMHO,  to be consistent we should fix in different issue

Reply:
agree

Reply:
agree